3570/3/rating_places/db/migrate/20190729131106_add_columns_comments_count_and_rating_cache_to_places.rb

```diff
@@ -0,0 +1,6 @@
+class AddColumnsCommentsCountAndRatingCacheToPlaces < ActiveRecord::Migration
+  def change
+    add_column :places, :comments_counter, :integer
+    change_column :places, :rating, :float
```

Comment:
As I see it's better to move out from this column too

Reply:
the column stores the average score